content/applications/inventory_and_mrp/quality/quality_management/quality_control_points.rst

```diff
@@ -1,89 +1,123 @@
+
+The :guilabel:`Control Frequency` field is set to one of three options that determine *how often* a
+new quality check is created:
+
+- :guilabel:`All`: a quality check is requested every time the conditions of the |QCP| are met.
+- :guilabel:`Randomly`: a quality check is randomly requested for a certain percentage of
+  operations, which can be specified in the :guilabel:`Every #% of Transfers` field that appears
+  below.
+- :guilabel:`Periodically`: a quality check is requested once every set period of time, which can be
+  specified by entering a numerical value and choosing either :guilabel:`Days`, :guilabel:`Weeks`,
+  or :guilabel:`Months` in the two fields that appear below.
+
+In the :guilabel:`Type` field, specify the type of quality check that should be performed. The
+method for processing quality checks created by the |QCP| depends upon the type of quality check
+selected:
+
+- :guilabel:`Instructions` checks provide specific instructions for how to complete the quality
+  check.
+- :guilabel:`Take a Picture` checks require a picture of the product be uploaded for later review by
+  the assigned quality team.
+- :guilabel:`Register Production` checks prompt manufacturing employees to confirm the quantity of
+  the product that was produced during the manufacturing operation.
+- :guilabel:`Pass - Fail` checks specify a criterion that products must meet for the check to pass.
+- :guilabel:`Measure` checks prompt employees to record a measurement of the product that must be
+  within a tolerance of a norm value for the check to pass.
+- :guilabel:`Worksheet` checks provide an interactive worksheet that must be filled out by the
+  employee processing the check.
+
+.. important::
+   An *Instructions* check is the same as a step on a work order for an MO. When a step is added to a
+   work order, Odoo stores it in the Quality app as a |QCP|. It is possible to manually create a
+   |QCP| with the *Instructions* check type, and even assign it to an operation other than
+   manufacturing, like receipts. However, when creating a control point specifically for quality
+   control purposes, using a different check type is probably more effective.
+
+In the :guilabel:`Team` field, specify the quality team that is responsible for managing the |QCP|
+and the quality checks it creates. If a specific quality team member is responsible for the |QCP|,
+select them in the :guilabel:`Responsible` field.
+
+The :guilabel:`Step Document` field has two options that specify the location of an instructional
+document detailing how to complete the quality checks created by the |QCP|. Select
+:guilabel:`Specific Page of Operation Worksheet` if the document is included with the instructional
+worksheet for the work order, then enter the page number in the :guilabel:`Worksheet Page` field
+that appears below. Select :guilabel:`Custom` if the document should be included in the
+:guilabel:`Instructions` tab at the bottom of the |QCP|.
```

Comment:
```suggestion
The :guilabel:`Step Document` field has two options that specify the location of an instructional
document detailing how to complete the quality checks created by the |QCP|.

Select :guilabel:`Specific Page of Operation Worksheet` if the document is included with the
instructional worksheet for the work order. Then, enter the page number in the :guilabel:`Worksheet
Page` field that appears below.

Select :guilabel:`Custom` if the document should be included in the :guilabel:`Instructions` tab at
the bottom of the |QCP|.
```
just breaking up an otherwise "chunky" paragraph to make it more digestible for the reader 👍